modules/core/CMakeLists.txt

```diff
@@ -60,6 +60,26 @@ if(CV_TRACE AND HAVE_ITT)
   add_definitions(-DOPENCV_WITH_ITT=1)
 endif()
 
+# https://github.com/opencv/opencv/issues/24145
+if(HAVE_IPP)
+  OCV_OPTION(OPENCV_IPP_ENABLE_ALL "Force to enable all ipp optimization" OFF)
```

Comment:
ipp => IPP

---

all => available ?

some implementations are still disabled due to accuracy or performance issues

Reply:
Thank you for your comment! 

I would like to make fix based on the Description of OPENCV_FORCE_3RDPARTY_BUILD.

> OCV_OPTION(OPENCV_IPP_ENABLE_ALL "Enable all OPENCV_IPP_ options at once" OFF)

https://docs.opencv.org/4.x/db/d05/tutorial_config_reference.html